config/application.rb

```diff
@@ -18,5 +18,11 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+
```

Comment:
why did we add these lines?

Reply:
The default code for devise already had them. These are some devise settings that can be overiden if there is ever a need for it. They were commented out thou, should i remove them?

Reply:
no we just want to remove the newly added line in this PR

Reply:
oh! i added that for the action mailer configuration.